src/ruby/README.md

```diff
@@ -0,0 +1,36 @@
+Steps to compile the extensions
+===============================
+* Install bundler by `gem install bundler`
+* Change directory to src/ruby, `cd src/ruby`
+* Execute `bundle install` this will install the gems required
+* Change to root directory `cd ../..` and Build symengine as a shared library
+and with Ruby by adding the flag `-DBUILD_SHARED_LIBS=on -DWITH_RUBY=yes`
+* Then cd to src/ruby again `cd src/ruby` and execute `rake install`
+to install the gem.
```

Comment:
`bundle install` installs the dependent gems. Shouldn't we do that before making the wrappers i.e. `cmake -DWITH_RUBY=yes .`